inspirehep/modules/search/query_factory.py

```diff
@@ -47,7 +49,7 @@
 def inspire_query_factory():
     """Create a parser returning Elastic Search DSL query instance."""
 
-    def invenio_query(pattern):
+    def invenio_query(pattern, index='records-hep'):
```

Comment:
@jmartinm for this function to be useful as a factory, should some initialization (e.g. the walkers below) being moved outside?